pkg/chartutil/create.go

```diff
@@ -39,36 +39,22 @@ import (
 var chartName = regexp.MustCompile("^[a-zA-Z0-9._-]+$")
 
 const (
-	// ChartfileName is the default Chart file name.
-	ChartfileName = "Chart.yaml"
-	// ValuesfileName is the default values file name.
-	ValuesfileName = "values.yaml"
-	// SchemafileName is the default values schema file name.
-	SchemafileName = "values.schema.json"
-	// TemplatesDir is the relative directory name for templates.
-	TemplatesDir = "templates"
-	// ChartsDir is the relative directory name for charts dependencies.
-	ChartsDir = "charts"
-	// TemplatesTestsDir is the relative directory name for tests.
-	TemplatesTestsDir = TemplatesDir + sep + "tests"
-	// IgnorefileName is the name of the Helm ignore file.
-	IgnorefileName = ".helmignore"
-	// IngressFileName is the name of the example ingress file.
-	IngressFileName = TemplatesDir + sep + "ingress.yaml"
-	// DeploymentName is the name of the example deployment file.
-	DeploymentName = TemplatesDir + sep + "deployment.yaml"
-	// ServiceName is the name of the example service file.
-	ServiceName = TemplatesDir + sep + "service.yaml"
-	// ServiceAccountName is the name of the example serviceaccount file.
-	ServiceAccountName = TemplatesDir + sep + "serviceaccount.yaml"
-	// HorizontalPodAutoscalerName is the name of the example hpa file.
-	HorizontalPodAutoscalerName = TemplatesDir + sep + "hpa.yaml"
-	// NotesName is the name of the example NOTES.txt file.
-	NotesName = TemplatesDir + sep + "NOTES.txt"
-	// HelpersName is the name of the example helpers file.
-	HelpersName = TemplatesDir + sep + "_helpers.tpl"
-	// TestConnectionName is the name of the example test file.
-	TestConnectionName = TemplatesTestsDir + sep + "test-connection.yaml"
+	ChartFilename                   = "Chart.yaml"
+	ValuesFilename                  = "values.yaml"
+	ValuesSchemaFilename            = "values.schema.json"
+	TemplatesDir                    = "templates"
+	ChartsDir                       = "charts"
+	TemplatesTestsDir               = TemplatesDir + sep + "tests"
+	IgnoreFilename                  = ".helmignore"
+	IngressFilename                 = TemplatesDir + sep + "ingress.yaml"
+	DeploymentFilename              = TemplatesDir + sep + "deployment.yaml"
+	ServiceFilename                 = TemplatesDir + sep + "service.yaml"
+	ServiceAccountFilename          = TemplatesDir + sep + "serviceaccount.yaml"
+	HorizontalPodAutoscalerFilename = TemplatesDir + sep + "hpa.yaml"
+	PodDisruptionBudgetFilename     = TemplatesDir + sep + "pdb.yaml"
+	NotesFilename                   = TemplatesDir + sep + "NOTES.txt"
+	HelpersFilename                 = TemplatesDir + sep + "_helpers.tpl"
+	TestConnectionFilename          = TemplatesTestsDir + sep + "test-connection.yaml"
```

Comment:
All of these had comments on them describing what they do. Some of that content matters. For example, `TemplateDir` is the _relative_ directory name. The relative nature is important. Can you revert the removing those.

Reply:
ok, but then shoudnt we just make the name self descriptive?

`TemplatesRelativeDir` ? Or is that a go anti pattern of sorts

Reply:
reverted, lets keep comments everywhere, dont tell uncle bob